charms/precise/clearwater-bono/lib/config_script

```diff
@@ -15,6 +15,8 @@ enum_server=
 
 # Apply new configuration
 home_domain=$(config-get zone)
+[ "$relation-name" != "cscf" ] || [ "$(relation-get public-address)" = "" ] || sprout_hostname=$(relation-get public-address)
```

Comment:
This isn't the right behaviour, is it? If I have `sprout_hostname=sprout.example.com` (resolving to all my Sprout nodes), and I spin up a new Sprout, it shouldn't change to `sprout_hostname=10.0.0.2`.